openedx/core/djangoapps/user_api/views.py

```diff
@@ -168,18 +168,19 @@ class RegistrationView(APIView):
         "city",
         "state",
         "country",
+        'privacy_policy',
         "gender",
         "year_of_birth",
         "level_of_education",
         "company",
         "title",
         "mailing_address",
         "goals",
-        "honor_code",
```

Comment:
Why did this change was done? These variables were reverted in the L182-183

Reply:
Done